### PR TITLE
[USING] rework tails setup

### PR DESCRIPTION
**Pull request by MaxHillebrand**
This ready for review branch reworks the tails setup.
I am not sure if the script is still valid for multiwallet support?

### CODE REVIEW

File: docs/using-wasabi/WasabiSetupTails.md [hunk 1]
```diff
@@ -113,34 +110,6 @@ After the first time you save a Wasabi session, your persistent storage will loo
...
-    |           |__ /Blocks
-    |__ /watch-only coldstorage B
-        |__ /.walletwasabi
-            |__ /client
-                |__ /Wallets
-                |__ /Blocks
-```
-
-This is only a minor example, tune it to your own needs.
-
```
Comment (MaxHillebrand) [inline docs/using-wasabi/WasabiSetupTails.md:143]: is this no longer needed with multiwallet?

**Comment by MaxHillebrand** [comment]
And does this script have to be updated for multiwallet support?

https://github.com/zkSNACKs/WasabiDoc/pull/744/files#diff-d26ca8632e23cf8a0b44b5f341a25e54R117-R157

**Comment by MaxHillebrand** [comment]
Which info are you talking about @RiccardoMasutti? 
The branch doesn't remove much, other than the script that I now think is obsolete.

In general, I am removing stuff so to declutter the docs.

**Comment by RiccardoMasutti** [comment]
Sorry I was talking about the folders for multiple wallet!
We don't need it anymore but I need to check the script is still valid

File: docs/using-wasabi/WasabiSetupTails.md [hunk 2]
```diff
@@ -68,11 +69,7 @@ Create a directory in your persistent with the same hierarchical structure, like
 
 After every session, when you’re done, navigate into `/Home/.walletwasabi/client` and copy the desired folders into your persistent directory.
 
-Generally, you’d like to save the `Wallets` and `Blocks` folders.
-
-The former contains your wallet information (e.g. keys, labels), while the latter includes the blocks needed to establish your balance.
-
-Could be also nice to save the `BitcoinStore` folder, which contains the [BIP 158](/using-wasabi/BIPs.md#bip-158-compact-block-filters-for-light-clients) block filters, so that you don’t have to download them again.
+Generally, you’d like to save the `Wallets` [keys, labels], `Blocks` [which are interesting to the wallet] and `BitcoinStore` [block filters] folders.
```
Comment (RiccardoMasutti) [inline docs/using-wasabi/WasabiSetupTails.md:72]:
```suggestion
Generally, you’d like to save the `Wallets` (keys, labels), `Blocks` (which are interesting to the wallet) and `BitcoinStore` (block filters) folders.
```

**Comment by RiccardoMasutti** [comment]
> Which info are you talking about @RiccardoMasutti?
> The branch doesn't remove much, other than the script that I now think is obsolete.
> 
> In general, I am removing stuff so to declutter the docs.

The multi wallet has nothing to do with giving the end user the possibility to use and manage Wasabi Wallet as he prefers. 

I think we should allow the user to decide how best he wants to use the software. Not everyone may want to use the multi wallet feature.

Maybe we should add a ::: tip to the top

**Comment by PulpCattel** [comment]
I've no time to review now, but I think the idea of multiple Wasabi folders in the persistent still apply. For example, the user could keep one .walletwasabi folder with few wallet files and another .walletwasabi with few other wallet files. In this case he would still benefit from the multi wallet feature and he could do that to, e.g., 100% avoid the risk of contaminating wallets that he wants to keep separate.

example:

    |__ /wallets A,B
    |   |__ /.walletwasabi
    |       |__ /client
    |           |__ /Wallets
    |           |__ /Blocks
    |__ /wallets C,D
        |__ /.walletwasabi
            |__ /client
                |__ /Wallets
                |__ /Blocks

File: docs/using-wasabi/WasabiSetupTails.md [hunk 3]
```diff
@@ -113,34 +110,6 @@ After the first time you save a Wasabi session, your persistent storage will loo
 
 To load your saved session, copy and paste the `.walletwasabi` folder into `/Home` before starting Wasabi.
```
Comment (yahiheb) [inline docs/using-wasabi/WasabiSetupTails.md:112]:
```suggestion

You can save multiple copies of `.walletwasabi` in your persistent, each with different data:

```sh
/Persistent
|__ /bitcoin-0.18.1            	   # Bitcoin Core launcher folder
|__ /Bitcoin                       # Bitcoin Core data folder
|__ /Wasabi                        # General Wasabi folder
    |__ /Wasabi-${currentVersion}.deb       # Wasabi installer
    |__ /BitcoinStore              # Filters (No need to keep multiple copies of them)
    |__ /Wallets A,B,C
    |   |__ /.walletwasabi
    |       |__ /client            # Here we save our wallet files and blocks
    |           |__ /Wallets
    |           |__ /Blocks
    |__ /Wallets D,E,F
    |   |__ /.walletwasabi
    |       |__ /client
    |           |__ /Wallets
    |           |__ /Blocks
    |__ /Wallets G,H,I
        |__ /.walletwasabi
            |__ /client
                |__ /Wallets
                |__ /Blocks
```

This is only a minor example, tune it to your own needs.

```

**Pull request merged**